src/openMVG/features/feature.hpp

```diff
@@ -114,6 +114,48 @@ class SIOPointFeature : public PointFeature {
   float orientation_;  // In radians.
 };
 
+class SIOPointFeature2 : public PointFeature {
```

Comment:
You are not using this class, so you can put back this file to his previous state.